app/assets/javascripts/application.js

```diff
@@ -22,6 +22,14 @@
 
   var module;
 
+  if(typeof console === 'undefined') {
```

Comment:
I've made a pull request to add it into the DM Toolkit (see https://github.com/alphagov/digitalmarketplace-frontend-toolkit/pull/164).

I'll raise another pull request to make this app use that version once merged.